src/helix-ui/index.js

```diff
@@ -0,0 +1,27 @@
+import { HXElement } from './elements/HXElement';
+import * as elements from './elements';
+
+function _defineElements () {
+    for (let attr in elements) {
+        elements[attr].$define();
+    }
+}
+
+function initialize () {
+    // If polyfills are used, initialize when polyfills are ready
+    // otherwise, initialize immediately
+    if (window.WebComponents) {
```

Comment:
Future-proof the bootstrap logic for when polyfills are no longer needed (also autodetection simplifies consumption)